docker/Dockerfile.s-aureus-pipeline

```diff
@@ -0,0 +1,50 @@
+# Note. We are not passing the pathogen into this dockerfile. We want to keep cyclomatic complexity low.
+# Code duplication is better than high cyclomatic complexity
```

Comment:
No. Please use Dockerfile.psga-pipeline instead of this file so that we have a common and unified way of building the main image.

Reply:
No that dockerfile with a magic parameter will lead to very unmaintainable code. 

Totally different things happen when you pass the parameter in, making the script very hard to test and reason about. 

Sometimes it's best to repeat code when you want to reduce the cyclomatic complexity.

Reply:
https://en.wikipedia.org/wiki/Cyclomatic_complexity

Reply:
When there are shared components that will all need updating at the same time I think there is a balance to be struck. This seems an almost complete copy, but if it works for now I am happy to have the discussion about how this should scale down the line, and proceed for now.

Reply:
It is a complete copy, apart from the removal of the argument.